go.mod

```diff
@@ -1,11 +1,11 @@
 module github.com/reddit/baseplate.go
 
-go 1.22
+go 1.22.0
```

Comment:
this change necessary?

Reply:
I believe it is, as thrift declared `1.22.0` in v0.21.0 https://github.com/apache/thrift/blob/v0.21.0/go.mod#L3, and only `1.22.0` is considered matching that, and `1.22` does not.

it won't be required once we update to 1.23 here, and in the next version of thrift I'll also remember to remove the patch version.